### DISCUSSION TITLE
Copy Files in Building process - Vite

### DISCUSSION THREAD

**Discussion by higuitadiaz** [quasarframework/quasar#13536] [CLI - PWA mode] [answered]
Hi, 

Quick question... I need to copy some static files in the building process according to mode (PWA/SPA/ SSR), are different statics files for each mode; with WebPack I used the CopyWebpackPlugin however, I don't know what/ how to use this function in vite. 

Could someone help me in this process?

**Comment by MilosPaunovic** [comment]
Looking for something like this: [https://www.npmjs.com/package/vite-plugin-copy](https://www.npmjs.com/package/vite-plugin-copy)?

**Reply by higuitadiaz** [reply]
Thanks for your reply!

I am new with Vite and I tested both plugins rollup-plugin-copy and vite-plugin-copy, but I will stay with rollup-plugin because it has more documentation.

x,

**Comment by metalsadman** [comment, marked as answer]
posting my answer here from discord, this one I've tested https://www.npmjs.com/package/rollup-plugin-copy
usage at `quasar.config.js`
```js
  build: {
    ...
    vitePlugins: ctx.prod
        ? [
            [
              'rollup-plugin-copy',
              {
                targets: [
                  { src: 'src/index.html', dest: 'dist/public' },
                  { src: ['assets/fonts/arial.woff', 'assets/fonts/arial.woff2'], dest: 'dist/public/fonts' },
                  { src: 'assets/images/**/*', dest: 'dist/public/images' }
                ],
                verbose: true
              }
            ],
            // other vite/rollup plugins
          ]
        : [
           // dev plugins
         ]

**Reply by higuitadiaz** [reply]
Thank you! it was very helpful for me.
